cdapython/fetch.py

```diff
@@ -1281,13 +1275,7 @@ def fetch_rows(
 
         data_source_query.l.node_type = 'column'
 
-        if table == 'somatic_mutation':
-            
```

Comment:
this bit needs to stay in (just rename the table) -- otherwise we end up trying to access the column `mutation_from_{item}`, which doesn't exist, because `mutation_data_source` doesn't exist -- we renamed and transformed the `mutation` table, but we're still pulling provenance data for mutations from `subject_data_source`.